lib/tasks/test_replication.rake

```diff
@@ -5,6 +5,20 @@ namespace :test do
   namespace :replication do
     desc "Setup environment for replication specs"
     task :setup => :initialize do
+      # write the darga migrations file
+      migrations_uri = URI('https://api.github.com/repos/ManageIQ/manageiq/contents/db/migrate?ref=darga')
+      json = Net::HTTP.get(migrations_uri)
+
+      migrations = JSON.parse(json).map do |h|
+        filename = h["path"].split("/")[-1]
+        filename.split('_')[0]
+      end
+
+      # eliminate any non-timestamp entries
+      migrations.keep_if { |timestamp| timestamp =~ /\d+/ }
+
+      File.write(Rails.root.join("spec/replication/util/data/darga_migrations"), migrations.sort.join("\n"))
+
```

Comment:
I try to leave code out of the rake tasks and move them into normal Ruby classes.  Can you move this into EvmTestSetupReplication or perhaps create a new class for it?